zerver/tests/test_subs.py

```diff
@@ -1642,7 +1642,7 @@ def test_realm_admin_remove_others_from_subbed_private_stream(self) -> None:
         are on.
         """
         result = self.attempt_unsubscribe_of_principal(
-            query_count=17,
+            query_count=33,
```

Comment:
Please include the reasoning for these changes to the query count in your commit message.